### PR TITLE
Updates copy to reflect the ArcGIS style guide concerning sign-in.

### PR DESCRIPTION
**Pull request by esreli**
TLDR;
"Sign in" is the verb, "Login" is the noun.

### CODE REVIEW

File: maps-app-ios/Maps App/App Notifications/LoginNotifications.swift [hunk 1]
```diff
@@ -40,12 +40,12 @@ extension MapsAppNotifications {
 
 // MARK: Internals
 extension MapsAppNotifications {
-    static func postLoginNotification(user:AGSPortalUser) {
+    static func postSignInNotification(user:AGSPortalUser) {
         NotificationCenter.default.post(name: MapsAppNotifications.Names.appLogin, object: mapsApp,
```
Comment (nixta) [inline maps-app-ios/Maps App/App Notifications/LoginNotifications.swift]: Should we also update the name of the notification definition to `MapsAppNotifications.Names.appSignIn`?

File: maps-app-ios/Maps App/App Notifications/LoginNotifications.swift [hunk 2]
```diff
@@ -40,12 +40,12 @@ extension MapsAppNotifications {
...
                                         userInfo: [LoginNotifications.userKey:user])
     }
 
-    static func postLogoutNotification() {
+    static func postSignOutNotification() {
         NotificationCenter.default.post(name: MapsAppNotifications.Names.appLogout, object: mapsApp)
```
Comment (nixta) [inline maps-app-ios/Maps App/App Notifications/LoginNotifications.swift]: Likewise should we also update the name of the notification definition to `MapsAppNotifications.Names.appSignOut`?

File: maps-app-ios/UI/Account View/Account.storyboard [hunk 3]
```diff
@@ -151,17 +144,17 @@
...
                             <containerView hidden="YES" opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="xSs-iQ-f4q" userLabel="Logged In Container View">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                                <rect key="frame" x="0.0" y="20" width="600" height="580"/>
                                 <connections>
                                     <segue destination="gYN-bY-eoQ" kind="embed" id="iGb-pJ-egr"/>
                                 </connections>
                             </containerView>
                             <containerView hidden="YES" opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Um7-OB-RNk" userLabel="Logged Out Container View">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                                <rect key="frame" x="0.0" y="20" width="600" height="580"/>
```
Comment (nixta) [inline maps-app-ios/UI/Account View/Account.storyboard]: Do these dimension changes matter? Or are they just a function of the displayed device in the storyboard editor? I think it's fine, but they do distract from the meat of the change. Don't worry about it if it's fine and doesn't bother you.

Comment (esreli) [reply maps-app-ios/UI/Account View/Account.storyboard]: > Or are they just a function of the displayed device in the storyboard editor?

Correct, this is an Xcode automatically generated change. This doesn't excuse, however, how it made it into this PR. I will be more diligent in the future!

Comment (nixta) [reply maps-app-ios/UI/Account View/Account.storyboard]: It's quite possible that this is a function of my original commits of the app including device-specific stuff that they shouldn't have.

**Comment by esreli** [comment]
@nixta perhaps i'm wrong here but i'm treating the Signin Notification as if it is a proper noun and with that, i've kept the name as-is. Your thoughts?

**Comment by nixta** [comment]
I think they should change. They notify of an action having completed, right? And if you look at the names of the wrapper-functions for generating them, there's a mismatch.

``` swift
static func postSignOutNotification() {
        NotificationCenter.default.post(name: MapsAppNotifications.Names.appLogout…
 ```

Also, while a `Login` is a noun according to the style guide, I can't think that a `Logout` is?

**Comment by esreli** [comment]
> They notify of an action having completed, right?

They do though inherently they are the name of the action, no?

> there's a mismatch

You have a point here.

**Pull request merged**